akka-cluster/src/main/protobuf/ClusterMessages.proto

```diff
@@ -55,6 +55,7 @@ message InitJoinAck {
 }
 
 message ConfigCheck {
+  // Don't use extensions, removed in proto3
   extensions 100 to 102;
```

Comment:
Considering that this wasn't being used, do we still need it? Or can it be removed on future releases?

Reply:
Yes I think we can as 100-102 have never been serialized but i'll do a quick test

Reply:
:+1: that would be great. I wondered about these but never used them in protobuf so kept quiet...

Reply:
I've tested this and is fine to remove. The generated files don't include anything for extensions, only when creating an extension it validates that the field numbers are marked as extensions in the parent type

Reply:
this isn't quite true, i was looking at protobuf3 generated code. Still works over the wire tho